src/main/resources/templates/db_scripts/oaipmh/revert_createOaiPmhViewFunction.sql

```diff
@@ -1,19 +1,18 @@
-drop index if exists item_metadata_updateddate_idx;
-
-drop index if exists holdings_record_metadata_updateddate_idx;
-
-drop index if exists audit_instance_createddate_idx;
-
-drop index if exists audit_holdings_record_createddate_idx;
-
-drop index if exists audit_item_createddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.item_metadata_updateddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.holdings_record_metadata_updateddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_instance_createddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_holdings_record_createddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_item_createddate_idx;
+
+drop index if exists ${myuniversity}_${mymodule}.instance_pmh_metadata_updateddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.item_pmh_metadata_updateddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.holdings_record_pmh_metadata_updateddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_instance_pmh_createddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_holdings_record_pmh_createddate_idx;
+drop index if exists ${myuniversity}_${mymodule}.audit_item_pmh_createddate_idx;
 
 drop function if exists ${myuniversity}_${mymodule}.pmh_view_function(timestamptz, timestamptz, bool, bool);
-
 drop function if exists ${myuniversity}_${mymodule}.strToTimestamp(text);
-
 drop function if exists ${myuniversity}_${mymodule}.dateOrMin(timestamptz);
-
 drop function if exists ${myuniversity}_${mymodule}.dateOrMax(timestamptz);
-
 drop function if exists ${myuniversity}_${mymodule}.getElectronicAccessName(val jsonb);
```

Comment:
createOaiPmhViewFunction.sql uses `create or replace function` therefore `drop function` is not needed.
If it were needed (for example when argument types have changed) it should be moved immediately before the `create or replace function` in createOaiPmhViewFunction.sql for readability.

Reply:
Do you think revert_createOaiPmhViewFunction.sql  is not needed at all if create or replace is used in our indexes and function?

Reply:
Revert script has been removed.